extensions/ql-vscode/src/view/data-extensions-editor/ModeledMethodsList.tsx

```diff
@@ -41,6 +41,7 @@ export const ModeledMethodsList = ({
           externalApiUsages={grouped[libraryName]}
           modeledMethods={modeledMethods}
           mode={mode}
+          hasUnsavedChanges={false}
```

Comment:
I've made this hidden by default, but to see it for testing just change this to true. You can use the "Data extensions editor" storybook to view it.

We could also add a storybook for the `LibraryRow` component, if that would be helpful.